config/routes.rb

```diff
@@ -2,15 +2,15 @@
   root 'amusement_parks#index'
   devise_for :users
 
-  resources :amusement_parks, only: [:index, :show] do
+  resources :amusement_parks, only: [:index, :show, :create, :new, :edit, :update] do
     resources :reviews, only: [:index, :create]
     resources :rides, only: [:show]
   end
 
 
   namespace :api do
     namespace :v1 do
-      resources :amusement_parks, only: [:index, :show] do
+      resources :amusement_parks, only: [:index, :show, :create, :new, :edit, :update] do
```

Comment:
This is fine and no need to update for merge, but at this point it may be better to use `except: :destroy`